tree/ntupleutil/v7/test/ntuple_inspector.cxx

```diff
@@ -133,17 +133,21 @@ TEST(RNTupleInspector, SizeCompressedInt)
 
       auto writeOptions = RNTupleWriteOptions();
       writeOptions.SetCompression(505);
+      // Ensure there are multiple clusters, so that we can test the size is correctly calculated when this is the case.
+      writeOptions.SetApproxZippedClusterSize(128);
       auto ntuple = RNTupleWriter::Recreate(std::move(model), "ntuple", fileGuard.GetPath(), writeOptions);
 
-      for (int32_t i = 0; i < 50; ++i) {
+      for (int32_t i = 0; i < 500; ++i) {
```

Comment:
I think a more direct way is to manually call `CommitCluster` in some iteration in the loop.